flow/connectors/clickhouse/clickhouse.go

```diff
@@ -77,24 +77,43 @@ func (c *ClickHouseConnector) ValidateCheck(ctx context.Context) error {
 	err := c.database.Exec(ctx, fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (
```

Comment:
```suggestion
	err := c.database.Exec(ctx, fmt.Sprintf(`CREATE TEMPORARY TABLE IF NOT EXISTS %s (
```
This would have table cleaned up when connector disposed